tests/integration/helm/helm-internal-auth_test.go

```diff
@@ -250,8 +255,20 @@ func (s *internalAuthHelmTest) TestHelmInstall() {
 
 				// get deployment
 				deployment := k8s.GetDeployment(subT, kubectlOptions, expected.name)
+
+				waitTime := 5 * time.Second
+				retries := 72
+
+				// `teams-api` pod does not start successfully on the first attempt.
```

Comment:
This conditional doesn't make a lot of sense to me. The total timeouts for both are 360 seconds

Reply:
Will open separate tickets for this